wiki/BioSQL.md

```diff
@@ -176,11 +190,11 @@ psql biosqldb < biosqldb-pg.sql
 
 Run *psql* and type enter *\\d <ENTER>* to see all the entities created.
 
-NCBI Taxonomy
--------------
+Populate the database With NCBI Taxonomy
+----------------------------------------
 
-The BioSQL package includes a perl script under
-scripts/load\_ncbi\_taxonomy.pl to download and update the taxonomy
+The BioSQL package includes a perl script under the
+**scripts** subdirectory named **load\_ncbi\_taxonomy.pl** that downloads and updates the taxonomy
```

Comment:
Use the double back-ticks for file names, ``load\_ncbi\_taxonomy.pl``, rather than double asterisk for bold.